### PR TITLE
Matched several functions

### PR DESCRIPTION
**Pull request by petrie911**
Matched Cutscene_ProcessCommands and func_8014D62C (z_message) from code. From overlays, BgSpdweb, ObjSpinyroll, EnArrow, and ObjSpidertent now fully match, EnSw is down to one nonmatching, and ObjectKankyo's nonmatching was improved.

### CODE REVIEW

**Review by AngheloAlf — APPROVED** [review]

**Review by tom-overton — APPROVED** [review]

File: src/overlays/actors/ovl_Bg_Spdweb/z_bg_spdweb.c [hunk 1]
```diff
@@ -350,23 +350,20 @@ void func_809CE4C8(BgSpdweb* this, PlayState* play) {
     CollisionCheck_SetAC(play, &play->colChkCtx, &this->collider.base);
 }
 
-#ifdef NON_MATCHING
 void func_809CE830(BgSpdweb* this, PlayState* play) {
```
Comment (EllipticEllipsis) [inline src/overlays/actors/ovl_Bg_Spdweb/z_bg_spdweb.c:353]: Think after all the work you did on understanding it you might as well sketch out some docs on this function.

File: src/code/z_demo.c [hunk 2]
```diff
@@ -1160,14 +1159,13 @@ void func_800ECD7C(CutsceneContext* csCtx, u8** cutscenePtr, s16 index) {
...
-    s32 totalEntries;
     s32 j;
-    u32 cmdType; // sp5C
     s32 pad;
-    s32 cmdEntries;       // sp58
+    s32 totalEntries;
+    u32 cmdType;    // sp5C
+    s32 cmdEntries; // sp58
+    s32 pad2;
     s32 cutsceneEndFrame; // sp50
```
Comment (hensldm) [inline src/code/z_demo.c:1168]:
```suggestion
    u32 cmdType;
    s32 cmdEntries;
    s32 pad2;
    s32 cutsceneEndFrame;
```

File: src/overlays/actors/ovl_Obj_Spinyroll/z_obj_spinyroll.h [hunk 3]
```diff
@@ -28,8 +28,8 @@ typedef struct {
 } ObjSpinyrollStruct; // size = 0x78
```
Comment (engineer124) [inline src/overlays/actors/ovl_Obj_Spinyroll/z_obj_spinyroll.h:28]: Could you delete this `ObjSpinyrollStruct` since it's now unused?

**Review by engineer124 — COMMENTED** [review]
Really great work!

File: src/overlays/actors/ovl_Obj_Spinyroll/z_obj_spinyroll.h [hunk 4]
```diff
@@ -24,22 +24,16 @@ typedef struct {
...
-} ObjSpinyrollStruct; // size = 0x78
-
-typedef struct {
-    /* 0x00 */ ObjSpinyrollStruct unk_00[2];
-    /* 0xF0 */ ObjSpinyrollStruct* unk_F0;
+    /* 0x00 */ ObjSpinyrollSubStruct unk_00[2][10];
+    /* 0xF0 */ ObjSpinyrollSubStruct* unk_F0;
     /* 0xF4 */ s32 unk_F4;
-} ObjSpinyrollStruct2; // size = 0xF8?
+} ObjSpinyrollStruct; // size = 0xF8?
```
Comment (hensldm) [inline src/overlays/actors/ovl_Obj_Spinyroll/z_obj_spinyroll.h:30]:
```suggestion
} ObjSpinyrollStruct; // size = 0xF8
```

**Review by hensldm — APPROVED** [review]
One more minor thing I noticed

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**